Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -184,3 +184,18 @@ public struct KlaviyoSDK {
         return false
     }
 }
+
+// MARK: - SPI Private methods
+
+// for internal use only
+
+@_spi(KlaviyoPrivate)
+extension KlaviyoSDK {
+    /// Fetch the active forms for the current user.
+    ///
+    /// - warning: For internal use only. The host app should not manually call this method, as
+    /// the logic for fetching and displaying forms will be handled internally within the SDK.
+    public func fetchForms() {
```

Comment:
Just generally curious - so before was this just not exposed to the test app? I remember you had another pr adding the actual network logic, so that never had logic to call the fetch? Or is this an additional way

Reply:
the previous PR put in the architecture to make the `fetch` call internally within the SDK, but didn't expose a public `fetch` method. The host app that consumes our SDK will never actually manually fetch the forms, so I didn't want to expose a public `fetch` method. For testing purposes, I had a local branch that temporarily exposed the `fetch` method publicly so I could wire it up in the test app, but I never tried to merge that for the reason stated above. 

After thinking some more about it, I figured that the approach in this PR works well; expose the `fetch` method, but hide it behind `@_spi` and document that it's for internal use only. Although the host app can technically call this method if they really wanted to, it's difficult because they need to import our SDK in an unconventional way, and they'll also see the documentation indicating that it's only for internal use. Also, if we decide later that we just don't want to have any possibility of the host app calling this method, we can simply revert this PR in the future, especially after we're done doing dev/testing work on IAF.